internal/stats/stats_service.go

```diff
@@ -460,6 +512,46 @@ func makePercentMD(prefix, metric string) metricDescriptor {
 	}
 }
 
+func makeNearCacheMSMD(ncName, metric string) metricDescriptor {
+	return metricDescriptor{
+		Prefix:             nearCacheDescriptorPrefix,
+		Metric:             metric,
+		Discriminator:      nearCacheDescriptorDiscriminator,
+		DiscriminatorValue: ncName,
+		HasUnit:            true,
+		Unit:               metricUnitMS,
+	}
+}
+
+func makeNearCacheCountMD(ncName, metric string) metricDescriptor {
+	return metricDescriptor{
+		Prefix:             nearCacheDescriptorPrefix,
+		Metric:             metric,
+		Discriminator:      nearCacheDescriptorDiscriminator,
+		DiscriminatorValue: ncName,
+		HasUnit:            true,
+		Unit:               metricUnitCount,
+	}
+}
```

Comment:
Alternative: send `Unit` as a parameter and remove one of the two methods.

Reply:
Added another helper at: https://github.com/hazelcast/hazelcast-go-client/pull/807/commits/0d49f98e13b8fc0d4843db1366d353fb6a96a174